_config.yml

```diff
@@ -1,6 +1,5 @@
 title: "Slick"
 description: "Scala Language-Integrated Connection Kit"
-baseurl: "http://slick.lightbend.com"
```

Comment:
I did a bit of research on this, and I think we don't need to have a `baseurl` configuration, unless we were serving from a subdirectory.

With it removed, Jekyll generates URLs to resources such as stylesheets in a form like "/resources/stylesheets/bootstrap.min.css" rather than absolute URLs like "http://slick.lightbend.com/resources/stylesheets/bootstrap.min.css" as on the current site.

I think this is also what prevents HTTPS from working (https://github.com/slick/slick/issues/2034).

See https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/